tests/scripts/stock/soft_delete_stock_test.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import patch
+
+import pytest
+
+from pcapi.model_creators.generic_creators import create_booking
+from pcapi.model_creators.generic_creators import create_offerer
+from pcapi.model_creators.generic_creators import create_payment
+from pcapi.model_creators.generic_creators import create_stock
+from pcapi.model_creators.generic_creators import create_user
+from pcapi.repository import repository
+from pcapi.scripts.stock.soft_delete_stock import soft_delete_stock
+
+
+class SoftDeleteStockTest:
+    @patch("pcapi.scripts.stock.soft_delete_stock.repository")
+    @pytest.mark.usefixtures("db_session")
+    def should_return_ko_if_at_least_one_booking_is_used(self, mock_repository, app):
+        # Given
+        user = create_user()
+        stock = create_stock(price=0)
+        booking = create_booking(is_used=True, stock=stock, user=user)
+
+        repository.save(booking)
```

Comment:
On essaie désormais de ne plus utiliser les fonctions de `generic_creators`, et plutôt d'utiliser les factories. Ici, ça donnerait quelque chose comme ça (avec un import `import pcapi.core.bookings.factories as bookings_factories`) :

```suggestion
        booking = bookings_factories.BookingFactory(isUsed=True)

        repository.save(booking)
```

Et tu peux accéder au stock avec `booking.stock`.

Idem dans les autres tests plus bas.